### ISSUE TITLE
Add a final WAIT for the lambda function to be updated

### ISSUE THREAD

**Issue by cfregly** [danilop/multimodal-chat#6]
final status of the function is "InProgress" when the script completes.

i'd prefer to have the script wait until the function been successfully updated before the script completes.

**Comment by danilop** [comment]
Good point. I think I tried that but couldn't find a correct wait state. It usually takes seconds so the worst it can happen is that a few invocations end up to the old image (if any). It should fail because there is a check on the state of the function.
